### ISSUE TITLE
[A11Y] Icon Documentation: Status message is not automatically announced by the screen reader.

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1790]
## Issue Details

Screen reader users MUST be informed when a web page launches an interstitial view, a progress indicator, or enters into a paused or busy state.

### Short Description

Status message is not automatically announced by the screen reader.

Element Description:
- No icon types found matching "arroe".

### Impact

- [ ] Blocker
- [ ] Critical
- [x] Serious
- [ ] Moderate
- [ ] Minor

### Steps To Reproduce:

Element location: Below the 'Filter' icons.

- In the 'Filter icon' field, enter 'Arroe'.
- No icon types found matching "arroe" message appears below search. 
- The status message is announced automatically only after entering another character (e.g. Arroew) in the search but is not announced as soon as it appears on the screen.

### Current Behavior:

The status message is only announced after an additional character is added after the initial status message appears.

### Expected Behavior:

The status message should be announced immediately when it appears.

---

### Auditor ID / URL

https://axeauditor.dequecloud.com/test-run/8b259ada-639e-11ef-962d-4f4fe35f0f31/issue/f3aebc3e-75b0-11ef-882c-0b3c67bdac7f?sortField=ordinal&sortDir=asc&filter%5Bseverity%5D=3&filter%5Btype%5D=issue&row=2

### Additional Details

<details>
<summary>Expand for extra context</summary>

- [x] Discovered during VPAT
- [ ] Found using NVDA
- [ ] Found using Chrome Screen Reader
- [ ] Found with special configuration 
</details>

---

## Screenshots 

![Image](https://github.com/user-attachments/assets/3e09e14d-fa11-4ed2-ad7d-033609e91459)

---

### 1. Perceivable

<details> 
<summary>Expand</summary>

#### 1.1 Text Alternatives

- [ ] 1.1.1 Non-text Content

#### 1.2 Time-based Media

- [ ] 1.2.1 Audio-only and Video-only (Prerecorded)
- [ ] 1.2.2 Captions (Prerecorded)
- [ ] 1.2.3 Audio Description or Media Alternative (Prerecorded)
- [ ] 1.2.4 Captions (Live)
- [ ] 1.2.5 Audio Description (Prerecorded)

#### 1.3 Info and Relationships

- [ ] 1.3.1 Info and Relationships
- [ ] 1.3.2 Meaningful Sequence
- [ ] 1.3.3 Sensory Characteristics
- [ ] 1.3.4 Orientation
- [ ] 1.3.5 Identify Input Purpose

#### 1.4 Distinguishable

- [ ] 1.4.1 Use of Color
- [ ] 1.4.2 Audio Control
- [ ] 1.4.3 Contrast (Minimum)
- [ ] 1.4.4 Resize text
- [ ] 1.4.5 Images of Text
- [ ] 1.4.10 Reflow
- [ ] 1.4.11 Non-text Contrast
- [ ] 1.4.12 Text Spacing
- [ ] 1.4.13 Content on Hover or Focus
</details>

### 2. Operable

<details> 
<summary>Expand</summary>

#### 2.1 Keyboard Accessible

- [ ] 2.1.1 Keyboard
- [ ] 2.1.2 No Keyboard Trap
- [ ] 2.1.4 Character Key Shortcuts

#### 2.2 Enough Time

- [ ] 2.2.1 Timing Adjustable
- [ ] 2.2.2 Pause, Stop, Hide

#### 2.3 Seizures and Physical Reactions

- [ ] 2.3.1 Three Flashes or Below Threshold

#### 2.4 Navigable

- [ ] 2.4.1 Bypass Blocks
- [ ] 2.4.2 Page Titled
- [ ] 2.4.3 Focus Order
- [ ] 2.4.4 Link Purpose (In Context)
- [ ] 2.4.5 Multiple Ways
- [ ] 2.4.6 Headings and Labels
- [ ] 2.4.7 Focus Visible

#### 2.5 Input Modalities

- [ ] 2.5.1 Pointer Gestures
- [ ] 2.5.2 Pointer Cancellation
- [ ] 2.5.3 Label in Name
- [ ] 2.5.4 Motion Actuation
</details>

### 3. Understandable

<details> 
<summary>Expand</summary>

#### 3.1 Readable

- [ ] 3.1.1 Language of Page
- [ ] 3.1.2 Language of Parts

#### 3.2 Predictable

- [ ] 3.2.1 On Focus
- [ ] 3.2.2 On Input
- [ ] 3.2.3 Consistent Navigation
- [ ] 3.2.4 Consistent Identification

#### 3.3 Input Assistance

- [ ] 3.3.1 Error Identification
- [ ] 3.3.2 Labels or Instructions
- [ ] 3.3.3 Error Suggestion
- [ ] 3.3.4 Error Prevention (Legal, Financial, Data)
</details>

### 4. Robust

<details>
<summary>Expand</summary>

#### 4.1 Compatible

- [ ] 4.1.1 Parsing
- [ ] 4.1.2 Name, Role, Value
- [x] 4.1.3 Status Messages
</details>
